packages/telemetry/lib/utils.js

```diff
@@ -0,0 +1,92 @@
+'use strict'
```

Comment:
this file should retain https://github.com/fastify/fastify-swagger/blob/master/LICENSE at the top. Please call it `formatParamUrl` instead.

Reply:
Done, PTAL @mcollina